paper/paper.bib

```diff
@@ -0,0 +1,70 @@
+@software{PyYAML:2024,
+  author = {{YAML and Python communities}},
+  title = {PyYAML},
+  year = {2024},
+  month = {Aug},
+  version = {latest},
+  url = {https://github.com/yaml/pyyaml},
+}
+
+@software{Click:2025,
+    author       = {{The click development team}},
+    title        = {Command Line Interface Creation Kit},
+    month        = {May},
+    year         = {2025},
+    version      = {latest},
+    url          = {https://github.com/pallets/click}
+}
+
+@software{lxml:2025,
+    author = {{The lxml development team}},
+    title = {lxml},
+    year = {2025},
+    month = {Jan},
+    version = {latest},
+    url = {https://github.com/lxml/lxml}
+}
+
+@article{Koennecke:2015,
+author = "K{\"{o}}nnecke, Mark and Akeroyd, Frederick A. and Bernstein, Herbert J. and Brewster, Aaron S. and Campbell, Stuart I. and Clausen, Bj{\"{o}}rn and Cottrell, Stephen and Hoffmann, Jens Uwe and Jemian, Pete R. and M{\"{a}}nnicke, David and Osborn, Raymond and Peterson, Peter F. and Richter, Tobias and Suzuki, Jiro and Watts, Benjamin and Wintersberger, Eugen and Wuttke, Joachim",
+title = "{The NeXus data format}",
+journal = "Journal of Applied Crystallography",
+year = "2015",
+volume = "48",
+number = "1",
+pages = "301--305",
+month = "Feb",
+doi = {10.1107/S1600576714027575},
+url = {https://doi.org/10.1107/S1600576714027575}
+}
+
+@misc{NIAC:2025,
+  title = {NIAC and NeXus Documentation},
+  author = {{NeXus International Advisory Committee}},
+  year = {2025},
+  month = {Aug},
+  url = {https://www.nexusformat.org},
+  note = {Accessed: 2025-08-01}
+}
+
+@techreport{YAML2009,
+  added-at = {2011-02-07T17:06:48.000+0100},
+  author = {Ben-Kiki, Oren and Evans, Clark and Ingerson, Brian},
+  biburl = {https://www.bibsonomy.org/bibtex/20b5235b9290624990cb61354f47c1ffd/voj},
+  institution = {YAML.org},
+  interhash = {d90ac6e7cf9c639bfa71929cd4928a54},
+  intrahash = {0b5235b9290624990cb61354f47c1ffd},
+  keywords = {dsl yaml},
+  month = {9},
+  timestamp = {2011-02-07T17:06:48.000+0100},
+  title = {{YAML} Ain't Markup Language ({YAML}) (tm) Version 1.2},
```

Comment:
we are not using version 1.2 so we should also add one refer for an earlier if existent

Reply:
Any idea, what version we are using?